network/test/src/sync.rs

```diff
@@ -0,0 +1,662 @@
+// Copyright 2017-2020 Parity Technologies (UK) Ltd.
```

Comment:
This file is unneeded

Reply:
(this functionality is tested in Substrate - Polkadot doesn't change the sync code)